config/environment.py

```diff
@@ -22,6 +22,9 @@ class Default(object):
     MAIL_PASSWORD = os.environ.get("MAIL_PASSWORD")
     MAIL_DEFAULT_SENDER = os.environ.get("MAIL_DEFAULT_SENDER")
 
+    # Default front end url
+    FRONTEND_BASE_URL = os.environ.get("FRONTEND_BASE_URL")
```

Comment:
Maybe we can do this instead? And then we can remove the override below, and it wont be necessary to set any env vars in development. If the env var set it will be used; otherwise localhost is used.

```suggestion
    FRONTEND_BASE_URL = os.environ.get("FRONTEND_BASE_URL", "http://localhost:3000")
```

Reply:
Using http here since we don't have https setup for dev env

Reply:
Didn't realize that ```os.environ.get``` accepted multiple parameters.